views/comp_avatar.ejs

```diff
@@ -0,0 +1,128 @@
+<%- include("partials/head") %>
+    <h1 id="avatar">Avatar</h1>
+    <p>
+        Avatar is a picture or component used to identify a user. <br>
+        Use classes in format <span class="example-in-article">"avatar-size"</span> to get avatar of
+        required size.
+        For example - "avatar-sm" for small size avatar.
+    </p>
+    <div class="component__container">
+        <img src="https://picsum.photos/72" alt="" class="avatar-xs">
+        <img src="https://picsum.photos/72" alt="" class="avatar-sm">
+        <img src="https://picsum.photos/72" alt="" class="avatar-md">
+        <img src="https://picsum.photos/72" alt="" class="avatar-lg">
+        <img src="https://picsum.photos/72" alt="" class="avatar-xl">
+    </div>
```

Comment:
alt tags should not be empty

Reply:
but content of alt tags depends on where i will be using them, here they are just for graphical representation ?